tests/test_mcts.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from alphazero.mcts import mcts, MCTSNode
+from alphazero import TicTacToe
+import torch
+
+
+class TestMCTS:
+    def test_tictactoe_first_step(self):
+        policy, _ = mcts(
```

Comment:
do we wanna check the Q value here too? or is this going to give us a random value that can't be tested?

Reply:
Sure we could, good point

Reply:
I think this requires too much unnecessary refactoring to get that information out so I'm going to wait with doing this. Remind me to create an issue for this